uaaextras/webapp.py

```diff
@@ -804,32 +706,7 @@ def logout():
 
         return redirect(url_for('index'))
 
-    threadEvent = threading.Event()
-
     app.config['SERVER_NAME'] = SERVER_NAME
     app.config['PREFERRED_URL_SCHEME'] = PREFERRED_URL_SCHEME
-    with app.app_context():
-        changeLink = url_for('change_password', _external=True)
```

Comment:
Looks like this variable get used elsewhere. I noticed due to failing tests--you can run locally with `python setup.py test`.